src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -21,5 +21,11 @@
     <type fullname="System.StartupHookProvider" feature="System.StartupHookProvider.IsSupported" featurevalue="false">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Resources.ManifestBasedResourceGroveler" feature="System.Resources.AllowReflectionForNonPrimitiveObjects" featurevalue="false">
+      <method signature="System.Boolean AllowUsingReflectionToLoadTypes()" body="stub" value="false" />
```

Comment:
> AllowReflectionForNonPrimitiveObjects vs. AllowUsingReflectionToLoadTypes

Is there a good reason for using two different names for the same thing? Should we use the same name everywhere?

Reply:
Good point, fixed.